wiki/en/Installation-for-Linux.md

```diff
@@ -54,22 +54,43 @@ For installers on other distributions, see their package managers and [Repology]
 
 ## Set up your hardware
 
+If no instance of a JACK server is running, one is normally started automatically.
+
+Jamulus will attach to a running JACK instance by name, using the `JACK_DEFAULT_SERVER` environment variable or `default` if the variable is not set.
+
+If your distribution uses JACK as the sound server, check out [how to configure JACK with QjackCtl](Installation-for-Linux#configure-jack-with-qjackctl).
+
+If your distribution uses PipeWire by default, check out the section on [configuring PipeWire](Installation-for-Linux#configure-pipewire).
+
 ### Configure JACK with QjackCtl
 
 Jamulus Clients need [JACK](https://jackaudio.org/) to run, but you need to configure that first. The recommended method is to use `QjackCtl`.
 
-1. Launch QjackCtl. You will see the **Qt JACK Control utility main page**
+1. Launch QjackCtl. You will see the **Qt JACK Control utility main page**.
 2. Configure your audio hardware as follows (the exact settings for JACK will depend on what your audio hardware is capable of):
 
-- Set the audio **Interface** to the one you want (there may be several in the list)
-- Set the **Sample Rate to 48000**
-- Set the **Frames/Period to 128** and Periods/Buffer at 2 at first
+- Set the audio **Interface** to the one you want (there may be several in the list).
+- Set the **Sample Rate to 48000**.
+- Set the **Frames/Period to 128** and Periods/Buffer at 2 at first.
 
-Restart JACK for the new settings to take effect
+Restart JACK for the new settings to take effect.
+
+### Configure PipeWire
+
+The Jamulus Client requires a JACK sound server (native PipeWire support has not yet been implemented).
+PipeWire provides a compatibility layer that emulates the JACK server, so when you launch the Jamulus Client, PîpeWire automatically exposes itself to it as a JACK server.
+You need to set your audio interface to the "pro-audio" profile and define PipeWire's "rate" and "quantum" parameters beforehand.
+To do so, follow these steps:
+1. open `pavucontrol` (the audio mixer) and choose the "pro-audio" profile in the configuration tab,
+2. In a terminal use the two following commands:
+```bash
+pw-metadata -n settings 0 clock.force-rate 48000
+pw-metadata -n settings 0 clock.force-quantum 128
+```
```

Comment:
Do the PipeWire "profiles" get defined is config files anywhere?  Can they be imported?  Can one profile be "based on" another?  Ideally, rather than providing shell commands, we'd provide a "Jamulus" profile and say "import this and make it active" -- the debian/ubuntu install could do it, making it even easier, then.

Reply:
PipeWire is using pulseaudio's profiles or AlsaRaw when using pro-audio profiles.
I don't where it is written.
As we don't do anything for JACK's users, I think we can not do the same for PipeWire's.

The PipeWire pro audio profile is explained [here](https://gitlab.freedesktop.org/pipewire/pipewire/-/wikis/FAQ#what-is-the-pro-audio-profile).
The FAQ is talking about a [native GUI tool](https://gitlab.freedesktop.org/pipewire/pipewire/-/wikis/FAQ#is-there-a-native-gui-tool-to-configure-pipewire)
There are also some info [here](https://gitlab.freedesktop.org/pipewire/pipewire/-/wikis/FAQ#pipewire-buffering-explained) about the buffering.

Reply:
It would be good to have a reference _directly_ to where `clock.force-rate` and `clock.force-quantum` are documented.

Reply:
Would a link to the page [jack-server](https://gitlab.freedesktop.org/pipewire/pipewire/-/wikis/Config-JACK#jack-server) be sufficient?

Reply:
done in https://github.com/jamulussoftware/jamuluswebsite/pull/980/commits/d1a3fbad686f22e2722074d92b5ea6ca601b886f